src/modules/visitor-space/utils/select/options.ts

```diff
@@ -4,5 +4,11 @@ export const getSelectValue = (
 	options: SelectOption[],
 	optionValue: string | undefined | null
 ): SelectOption | undefined => {
+	if (optionValue === undefined) {
+		return {
+			label: 'Select...',
```

Comment:
Add translation for this.

Reply:
![image](https://user-images.githubusercontent.com/113892598/236150916-29cbee5b-38ce-4ab7-b885-eed5191f6a10.png)
Can't use useTranslation because it's not a react component.